Source/Manager/Identifiers/EmailAddress.swift

```diff
@@ -54,3 +54,9 @@ public struct EmailAddress: IdentifierProtocol {
         return identifier
     }
 }
+
+extension EmailAddress: Equatable {
+    public static func == (lhs: EmailAddress, rhs: EmailAddress) -> Bool {
+        return lhs.normalizedString == rhs.normalizedString
+    }
+}
```

Comment:
do you need this?

it seems the only property of `EmailAddress` is `emailAddress` which is a string that is already `Equatable`, then you just declare `EmailAddress: Equatable` you don't need to define `==` then

Reply:
Ah nice catch! Thanks!